contracts/beacon/IVoucher.sol

```diff
@@ -38,4 +38,5 @@ interface IVoucher {
     function getBalance() external view returns (uint256);
     function isAccountAuthorized(address account) external view returns (bool);
     function getSponsoredAmount(bytes32 dealId) external view returns (uint256);
+    function isRefundedTask(bytes32 taskId) external view returns (bool);
```

Comment:
switch places with `getSponsoredAmount`

Reply:
Not sure to understand why (?):
```
    struct VoucherStorage {
        address _owner;
        address _voucherHub;
        uint256 _expiration;
        uint256 _type;
        mapping(address => bool) _authorizedAccounts;
        mapping(bytes32 dealId => uint256) _sponsoredAmounts;
        // Save refunded tasks to disable replay attacks.
        mapping(bytes32 taskId => bool) _refundedTasks;
    }
```

Reply:
We try to order functions by visibility first, then by storage order (for getter), wdt?

Reply:
For the order: Yes mb thank you for highlighting this point